app/controllers/admin/location_types_controller.rb

```diff
@@ -0,0 +1,66 @@
+module Admin
+  class LocationTypesController < Admin::ApplicationController
+    before_action :load_location_type, except: %i(new index create search)
+
+    def index
+      @location_types = LocationType.order(:name).page(params[:page])
+        .per Settings.controllers.admin.location_types.pag
+    end
+
+    def search
+      @location_types = SearchQuery.new(LocationType,search_params,Settings.search_fields.location_type).all.page(params[:page]).per Settings.controllers.admin.location_types.pag
+      render :index
+    end
+
+    def show; end
+
+    def new
+      @location_type = LocationType.new
+    end
+
+    def create
+      @location_type = LocationType.new location_type_params
+      if @location_type.save
+        flash[:success] = t "controllers.admin.location_types.create_success"
+        redirect_to request.referer
+      else
+        render :new
+      end
+    end
+
+    def edit; end
+
+    def update
+      if @location_type.update location_type_params
+        flash[:success] = t "controllers.admin.location_types.update_success"
```

Comment:
```suggestion
        flash[:success] = t ".update_success"
```
Rồi viết I18n sao cho nó chạy dc nhé, em dùng địa chỉ tuyệt đối này dài quá